src/runtime_src/xdp/profile/database/static_info/filetypes/aie_control_config_filetype.cpp

```diff
@@ -256,11 +257,15 @@ AIEControlConfigFiletype::getInterfaceTiles(const std::string& graphName,
         // Make sure column is within specified range (if specified)
         if (useColumn && !((minCol <= shimCol) && (shimCol <= maxCol)))
             continue;
-        // Make sure channel number is same as specified (GMIO only)
-        if ((type == io_type::GMIO) && (channelId >= 0) && (channelId != io.second.channelNum)) {
+
+        // Make sure stream/channel number is as specified
+        // NOTE: For GMIO we use DMA channel number; for PLIO, we use the SOUTH location
+        uint8_t idToCheck = (type == io_type::GMIO) ? channelNum : streamId;
+        if ((specifiedId >= 0) && (specifiedId != idToCheck)) {
+            std::string idType = (type == io_type::GMIO) ? "channel" : "stream";
             std::stringstream msg;
-            msg << "Specified channel ID " << +channelId << "doesn't match for interface column "
-                << +shimCol <<" and stream ID " << +streamId;
+            msg << "Specified " << idType << " ID " << +specifiedId << "doesn't match for "
```

Comment:
can we remove this xrt message altogether? In some instances while iterating over all IOs, it generates log for all not matched channelIDs.

Reply:
removed it